src/DotNest.Core.SDK.Web/Program.cs

```diff
@@ -1,13 +1,16 @@
-using NLog.Web;
+using Lombiq.Hosting.Tenants.Management.Extensions;
+using OrchardCore.Logging;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Logging.ClearProviders();
-builder.Host.UseNLog();
+builder.Host.UseNLogHost();
 
 var configuration = builder.Configuration;
 
-builder.Services.AddOrchardCms(orchardCoreBuilder =>
+builder.Services
+    .AddSingleton(configuration)
+    .AddOrchardCms(orchardCoreBuilder =>
 {
```

Comment:
Style this section like this: https://github.com/Lombiq/DotNest-Tenants-Core/blob/dev/src/DotNest.Tenants.Core.Web/Program.cs#L19

(So indent the curly brackets.)